src/main/java/Event.java

```diff
@@ -7,6 +7,16 @@
     protected LocalDate by;
     public String type;
```

Comment:
Maybe this could be protected also? I understand maybe it's public for accessibility reasons, but would it be good practice to make it protected and then have a getter method for the type of task?